content/monitors/notifications.md

```diff
@@ -309,6 +309,38 @@ Use `@here` in the monitor message to notify everybody in a given Hipchat channe
 {{% /tab %}}
 {{< /tabs >}}
 
+## Test monitor notifications
+
+After you define your monitor, you can test what your monitor's notification would look like in any applicable state. You can test what's specified in the "Say What's Happening" field by selecting "Select all" or selecting specific states.
+
+{{< img src="monitors/notifications/test-notif-select.png" alt="Test the notifications for this monitor" responsive="true" style="width:50%;" >}}
+
+You can only test states that are available in the monitor’s configuration and test for thresholds specified in the alerting conditions. Recovery thresholds are an exception, as Datadog sends a recovery notification once the monitor is no longer in alert or has no warn conditions.
+
```

Comment:
Can we link recovery thresholds to this: https://docs.datadoghq.com/monitors/faq/what-are-recovery-thresholds/ 

This way a user can continue reading about recovery thresholds since this input is optional

Reply:
👍